exercises/follow_line/web-template/brain.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import time
+import threading
+import multiprocessing
+import sys
+from datetime import datetime
+import re
+import json
+import traceback
+import imp
+
+import rospy
+from std_srvs.srv import Empty
+import cv2
+
+from user_functions import GUIFunctions, HALFunctions
+from console import start_console, close_console
+
+from shared.value import SharedValue
+
+# The brain process class
+class BrainProcess(multiprocessing.Process):
+    def __init__(self, code, exit_signal):
+        super(BrainProcess, self).__init__()
+
+        # Initialize exit signal
+        self.exit_signal = exit_signal
+
+        # Function definitions for users to use
+        self.hal = HALFunctions()
+        self.gui = GUIFunctions()
+
+        # Time variables
+        self.time_cycle = SharedValue('brain_time_cycle')
+        self.ideal_cycle = SharedValue('brain_ideal_cycle')
+        self.iteration_counter = 0
+
+        # Get the sequential and iterative code
+        # Something wrong over here! The code is reversing
+        # Found a solution but could not find the reason for this
+        self.sequential_code = code[1]
```

Comment:
I know this PR is closed but since I found this comment, 
https://github.com/JdeRobot/RoboticsAcademy/blob/768d8bfab0c42fdcfcb59aec55773ea60fda7213/exercises/follow_line/web-template/exercise.py#L99

Could this be the reason?

Reply:
Yes! This is the reason.

Thanks for pointing out! I'll make this change in one of other PRs.